ord_app/api/editor/reactions.py

```diff
@@ -94,3 +95,12 @@ def delete_reaction(user_id: str, dataset_name: str, index: int):
         del dataset.reactions[index]
         add_dataset(user_id, dataset, cursor)
     return Response(status_code=200)
+
+
+@router.post("/summarize_reaction")
+async def summarize_reaction(request: Request):
+    """Returns summary information for the reaction card view."""
+    # TODO(skearnes): Implement something useful here.
+    reaction = Reaction.FromString(await request.body())
+    del reaction  # Unused.
```

Comment:
Minor: it's not necessary, Python's GC will delete this object on its own.

Reply:
This is about lint; see "Unused" here: https://google.github.io/styleguide/pyguide.html#214-decision